src/Controller/EnMarche/ProcurationController.php

```diff
@@ -37,11 +37,13 @@ public function landingAction(Request $request): Response
      */
     public function indexAction(Request $request): Response
     {
-        $this->getProcurationFlow()->reset();
+        $procurationSession = $this->get(ProcurationRequestSession::class);
+        // clear previous session if any
```

Comment:
Why ?
By the way the comment does not bring more information that the method name.

Reply:
My bad, is this better now?